c/tests/assembly/CMakeLists.txt

```diff
@@ -18,7 +18,7 @@
 # specific language governing permissions and limitations
 # under the License.
 
-cmake_minimum_required(VERSION 3.15)
+cmake_minimum_required(VERSION 3.10)
```

Comment:
Ubuntu 18.04 ships CMake 3.10. Thankfully, we're not using any of later features.